siembol/values.yaml

```diff
@@ -1,28 +0,0 @@
-nginx-ingress:
```

Comment:
This has nothing to do with the upgrade of Storm 😅

Reply:
Sure, I can remove these.  Was just cleaning up some cruft...

Reply:
We should definitely clean those up. I'll open a separate PR for it.

Reply:
I created PR #10 to clean up the cruft, and I force pushed on this branch to clean up the clean up.